packages/tupaia-web/src/features/DashboardItem/EnlargedDashboardItem.tsx

```diff
@@ -23,28 +30,114 @@ const Container = styled.div`
   .recharts-responsive-container {
     min-height: 22.5rem;
   }
+  .recharts-cartesian-axis-tick {
+    font-size: 0.875rem;
+  }
+`;
+
+const Title = styled(Typography).attrs({
+  variant: 'h2',
+})`
+  font-size: 1.25rem;
+  font-weight: ${({ theme }) => theme.typography.fontWeightMedium};
+  text-align: center;
+  margin: 0.3rem 0 1rem 0;
+  line-height: 1.4;
+`;
+
+const TitleWrapper = styled.div`
```

Comment:
What do you think about making a little util component for this called FlexRow or something?